web-ng/src/app/components/map/map.component.ts

```diff
@@ -81,13 +83,39 @@ export class MapComponent implements AfterViewInit, OnDestroy {
   }
 
   ngAfterViewInit() {
+    const dummyProject: Project = new Project(
+      '',
+      StringMap({ en: 'title' }),
+      StringMap({ en: 'description' }),
+      Map({}),
+      Map({})
+    );
```

Comment:
Instead of creating placeholder project and features here for initial state, can we use `switchMap` to chain the streams as we do elsewhere? That might look something like:
```js
this.activeProject$
  .pipe(switchMap(project => this.features$.subscribe(features => doStuff...)));
```

Reply:
I did this because I was using `pairwise`, removed since now not using `pairwise` anymore.